### DISCUSSION TITLE
PackIcon does not see the icons

### DISCUSSION THREAD

**Discussion by Frillaut** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3353] [Q&A] [answered]
Hi!
I can't find these two icons. In the demonstrative application, they are visible... Is that a mistake?

```
<materialDesign:PackIcon Kind="LinkCircle" />
<materialDesign:PackIcon Kind="LinkCircleOutline" />
```

**Comment by Keboo** [comment]
The icons for this library are automatically generated from here: https://pictogrammers.com/library/mdi/
We also keep a running list of icon changes each release: https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/PackIcon-ReleaseNotes

I don't see any icons with that name, are those icon names you have previously been using?

**Reply by Frillaut** [reply]
No. I haven't used them before. I used the MaterialDesignDemo app. And there I found them.

![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/116347513/f0cbb41d-774b-4d48-873d-ba21c87d1f22)

**Reply by nicolaihenriksen** [reply]
@Frillaut @Keboo I am a little confused. Where is it that you don't see these? They are definitely part of the latest on master:
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/44bb9ba470aeffc642b9b64004d9c224164d95dc/MaterialDesignThemes.Wpf/PackIconKind.cs#L7893-L7894

And therefore also available in the demo app.

**Reply by Frillaut** [reply]
In version 4.9.0 for WPF, there are no such icons.

**Reply by nicolaihenriksen** [reply, marked as answer]
@Frillaut Ok, they are probably recent additions then. They are probably part of the 5.0 preview package I would assume?

**Reply by Keboo** [reply]
Ah that is what I get for trusting GitHub search rather than opening VS. Thank you @nicolaihenriksen
